boards/arm/rak4631_nrf52840/doc/index.rst

```diff
@@ -0,0 +1,153 @@
+.. _rak4631_nrf52840:
+
+RAK4631
+#######
+
+Overview
+********
+
+RAK4631 is a WisBlock Core module for RAK WisBlock.
+It extends the WisBlock series with a powerful
+Nordic nRF52840 MCU that supports Bluetooth 5.0
+(Bluetooth Low Energy) and the newest LoRa transceiver
+from Semtech, the SX1262. The Semtech SX1262 has compared
+to the older SX127x series a lower power consumption at
+the same TX power. This makes the RAK4631 an ultra-low
+power communication solution. RAK4631 can be comfortably
+programmed with ZephyrRTOS.
+
+.. image:: img/rak4631-front-parts.jpg
+     :width: 500px
+     :align: center
+     :alt: RAK4631-NRF52840
+
+Hardware
+********
+
+To use a RAK4631, you need at least a WisBlock Base
+to plug the module in. WisBlock Base is the power
+supply for the RAK4631 module and has the
+programming/debug interface.
+
+- nRF52840 ARM Cortex-M4F Processor
+- 64 MHz CPU clock
+- 1 Micro-AB USB OTG host/device
+- Semtech SX1262 low power high range LoRa transceiver
+- iPEX connectors for the LORA antenna and BLE antenna.
+- Multiple interfaces, I2C, UART, GPIO, ADC
+- 2 user LEDs on RAK5005 mother Board
+- Powered by either Micro USB, 3.7V rechargeable battery or a 5V Solar Panel Port
+
+Supported Features
+==================
+
+The rak4631_nrf52840 board configuration supports the following hardware features:
+
++-----------+------------+----------------------+
+| Interface | Controller | Driver/Component     |
++===========+============+======================+
+| ADC       | on-chip    | adc                  |
++-----------+------------+----------------------+
+| CLOCK     | on-chip    | clock_control        |
++-----------+------------+----------------------+
+| FLASH     | on-chip    | flash                |
++-----------+------------+----------------------+
+| GPIO      | on-chip    | gpio                 |
++-----------+------------+----------------------+
+| I2C(M)    | on-chip    | i2c                  |
++-----------+------------+----------------------+
+| MPU       | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| NVIC      | on-chip    | arch/arm             |
++-----------+------------+----------------------+
+| PWM       | on-chip    | pwm                  |
++-----------+------------+----------------------+
+| RADIO     | on-chip    | Bluetooth,           |
+|           |            | ieee802154           |
++-----------+------------+----------------------+
+| RTC       | on-chip    | system clock         |
++-----------+------------+----------------------+
+| SPI(M/S)  | on-chip    | spi                  |
++-----------+------------+----------------------+
+| UART      | on-chip    | serial               |
++-----------+------------+----------------------+
+| USB       | on-chip    | usb                  |
++-----------+------------+----------------------+
+| WDT       | on-chip    | watchdog             |
++-----------+------------+----------------------+
```

Comment:
Can we add LoRa here as it is one of the supported features?

Reply:
yes of course, I will add it